### ISSUE TITLE
Pointy graticules

### ISSUE THREAD

**Issue by Maschette** [hypertidy/graticule#14]
when using tiles over long distances they become quite pointy, Is there a way to make them properly curved? 
```
x<-graticule::graticule(lons = c(95, 135, 175),lats = c(-42, -39), tiles = TRUE, proj = raster::projection(Bathy))

SOmap()
plot(x, add=TRUE)
```
![image](https://user-images.githubusercontent.com/14663215/54891745-a6e40100-4f02-11e9-9c09-115ed2c3994c.png)

**Comment by mdsumner** [comment]
A sad problem with graticule, here's a workaround

```R
curvy_g <- function(lons, lats, proj = NULL, incr = 1) {
  if (is.null(proj)) stop("'proj' must be supplied")
  r <- raster::raster(raster::extent(range(lons), range(lats)), 
         ncols = length(lons) - 1, nrows = length(lats) - 1)
  r[] <- 0
  ll_crs <- "+proj=longlat +ellps=WGS84 +datum=WGS84 +no_defs +towgs84=0,0,0"
  out <- sf::as_Spatial(sf::st_segmentize( spex::polygonize(r), dfMaxLength = incr))
  sp::proj4string(out) <- sp::CRS(ll_crs)
  sp::spTransform(out, proj)
}

prj <- "+proj=stere +lat_0=-90 +lat_ts=-70 +lon_0=0 +k=1 +x_0=0 +y_0=0 +a=6378273 +b=6356889.449 +units=m +no_defs"
g <- curvy_g(lons = c(95, 135, 175), 
        lats = c(-42, -39), 
        proj = prj
)
plot(g)

```

Set `incr` to something finer as needed (set to 1-degree by default).

**Comment by mdsumner** [comment]
See a better fix coming from #15
